sdk/core/azure_core_amqp/Cargo.toml

```diff
@@ -49,6 +53,7 @@ fe2o3_amqp = [
   "serde_amqp",
   "serde_bytes",
 ]
+socks5 = ["dep:tokio-socks", "dep:native-tls", "dep:tokio-native-tls"]
```

Comment:
You also need to add this to the docs.rs metadata table below or it won't be documented. We intentionally don't document the others as they are meant more for internal testing / development.

Reply:
added